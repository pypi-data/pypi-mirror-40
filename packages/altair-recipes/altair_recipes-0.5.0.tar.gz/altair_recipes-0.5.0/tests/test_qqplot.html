
<div class="highlight"><pre><span></span><span class="kn">import</span> <span class="nn">altair_recipes</span> <span class="kn">as</span> <span class="nn">ar</span>
<span class="kn">from</span> <span class="nn">altair_recipes.common</span> <span class="kn">import</span> <span class="n">viz_reg_test</span>
<span class="kn">from</span> <span class="nn">altair_recipes.display_altair</span> <span class="kn">import</span> <span class="n">show_test</span>
<span class="kn">import</span> <span class="nn">numpy</span> <span class="kn">as</span> <span class="nn">np</span>
<span class="kn">import</span> <span class="nn">pandas</span> <span class="kn">as</span> <span class="nn">pd</span>
</pre></div>


<script src="https://cdn.jsdelivr.net/npm/vega@3"></script>
<script src="https://cdn.jsdelivr.net/npm/vega-lite@2"></script>
<script src="https://cdn.jsdelivr.net/npm/vega-embed@3"></script>

<h2>Qqplot</h2>




<div class="highlight"><pre><span></span><span class="nd">@viz_reg_test</span>
<span class="k">def</span> <span class="nf">test_qqplot</span><span class="p">():</span>

    <span class="n">df</span> <span class="o">=</span> <span class="n">pd</span><span class="o">.</span><span class="n">DataFrame</span><span class="p">(</span>
        <span class="p">{</span>
            <span class="s2">&quot;Trial A&quot;</span><span class="p">:</span> <span class="n">np</span><span class="o">.</span><span class="n">random</span><span class="o">.</span><span class="n">normal</span><span class="p">(</span><span class="mi">0</span><span class="p">,</span> <span class="mf">0.8</span><span class="p">,</span> <span class="mi">1000</span><span class="p">),</span>
            <span class="s2">&quot;Trial B&quot;</span><span class="p">:</span> <span class="n">np</span><span class="o">.</span><span class="n">random</span><span class="o">.</span><span class="n">normal</span><span class="p">(</span><span class="o">-</span><span class="mi">2</span><span class="p">,</span> <span class="mi">1</span><span class="p">,</span> <span class="mi">1000</span><span class="p">),</span>
            <span class="s2">&quot;Trial C&quot;</span><span class="p">:</span> <span class="n">np</span><span class="o">.</span><span class="n">random</span><span class="o">.</span><span class="n">uniform</span><span class="p">(</span><span class="mi">3</span><span class="p">,</span> <span class="mi">2</span><span class="p">,</span> <span class="mi">1000</span><span class="p">),</span>
        <span class="p">}</span>
    <span class="p">)</span>
    <span class="k">return</span> <span class="n">ar</span><span class="o">.</span><span class="n">qqplot</span><span class="p">(</span><span class="n">df</span><span class="p">,</span> <span class="n">x</span><span class="o">=</span><span class="s2">&quot;Trial A&quot;</span><span class="p">,</span> <span class="n">y</span><span class="o">=</span><span class="s2">&quot;Trial C&quot;</span><span class="p">)</span>


<span class="n">show_test</span><span class="p">(</span><span class="n">test_qqplot</span><span class="p">)</span>
</pre></div>


 <div id="A8183306fe917e1899f04d6633826bf78935eb3f10a5a31ad992722a3eabe4c87"></div>
  <script type="text/javascript">
    var spec = {
  "$schema": "https://vega.github.io/schema/vega-lite/v2.6.0.json",
  "config": {
    "view": {
      "height": 300,
      "width": 400
    }
  },
  "data": {
    "name": "data-b9c4aead55e3322b35e64eef74b39327"
  },
  "datasets": {
    "data-b9c4aead55e3322b35e64eef74b39327": [
      {
        "Trial A": -2.503987262261268,
        "Trial B": -4.90392241606464,
        "Trial C": 2.000532153972431
      },
      {
        "Trial A": -1.858947052184403,
        "Trial B": -4.294224601631242,
        "Trial C": 2.0229877208932385
      },
      {
        "Trial A": -1.5590333050173182,
        "Trial B": -4.019260620756446,
        "Trial C": 2.0326871019662276
      },
      {
        "Trial A": -1.462916412726826,
        "Trial B": -3.8428050238031854,
        "Trial C": 2.0455016448582666
      },
      {
        "Trial A": -1.3701285563392525,
        "Trial B": -3.6910092132999206,
        "Trial C": 2.0540598177192417
      },
      {
        "Trial A": -1.26301671255689,
        "Trial B": -3.6063291023281274,
        "Trial C": 2.061322218104995
      },
      {
        "Trial A": -1.2199209819219066,
        "Trial B": -3.524870764176491,
        "Trial C": 2.0785690904232186
      },
      {
        "Trial A": -1.1858859200451275,
        "Trial B": -3.4190352742009447,
        "Trial C": 2.0875840331396307
      },
      {
        "Trial A": -1.1411578266862328,
        "Trial B": -3.33934994895971,
        "Trial C": 2.094842718119722
      },
      {
        "Trial A": -1.0923165720984442,
        "Trial B": -3.238231116042695,
        "Trial C": 2.10620284351348
      },
      {
        "Trial A": -1.0283973545763812,
        "Trial B": -3.206309822258897,
        "Trial C": 2.111785389065272
      },
      {
        "Trial A": -1.001736590710582,
        "Trial B": -3.1750801346277893,
        "Trial C": 2.1202837528036156
      },
      {
        "Trial A": -0.942113194408969,
        "Trial B": -3.1220497264253857,
        "Trial C": 2.1297967570286076
      },
      {
        "Trial A": -0.890973671123184,
        "Trial B": -3.0644586807131975,
        "Trial C": 2.1382126404624744
      },
      {
        "Trial A": -0.8475694189531534,
        "Trial B": -3.019467581880705,
        "Trial C": 2.145341979255685
      },
      {
        "Trial A": -0.8249240554491132,
        "Trial B": -2.962926291226879,
        "Trial C": 2.1578093793563213
      },
      {
        "Trial A": -0.7736871985365446,
        "Trial B": -2.909664489887793,
        "Trial C": 2.172065378332868
      },
      {
        "Trial A": -0.7260549470706815,
        "Trial B": -2.8773675852486584,
        "Trial C": 2.1774036340028995
      },
      {
        "Trial A": -0.691978994594258,
        "Trial B": -2.844849335329879,
        "Trial C": 2.189892577548054
      },
      {
        "Trial A": -0.661438524703978,
        "Trial B": -2.825785243542171,
        "Trial C": 2.1981417825490945
      },
      {
        "Trial A": -0.6265863746017574,
        "Trial B": -2.793993749904662,
        "Trial C": 2.2081491886021376
      },
      {
        "Trial A": -0.6106137180567885,
        "Trial B": -2.7532621192119437,
        "Trial C": 2.2140914336227997
      },
      {
        "Trial A": -0.5881940114533255,
        "Trial B": -2.7273298589342456,
        "Trial C": 2.2251313634066694
      },
      {
        "Trial A": -0.5630563690460455,
        "Trial B": -2.7073164251869883,
        "Trial C": 2.2349121397186167
      },
      {
        "Trial A": -0.5432174000403499,
        "Trial B": -2.683596859713727,
        "Trial C": 2.2427276290862888
      },
      {
        "Trial A": -0.528544583026506,
        "Trial B": -2.6556640558860947,
        "Trial C": 2.2497980633691186
      },
      {
        "Trial A": -0.4924363911811489,
        "Trial B": -2.6255789441711443,
        "Trial C": 2.2593296703204335
      },
      {
        "Trial A": -0.46165589221451203,
        "Trial B": -2.6027738963041385,
        "Trial C": 2.267039859612697
      },
      {
        "Trial A": -0.4409321905614168,
        "Trial B": -2.570154680094415,
        "Trial C": 2.279715751210909
      },
      {
        "Trial A": -0.4290202448020787,
        "Trial B": -2.552660498966847,
        "Trial C": 2.2860463054858498
      },
      {
        "Trial A": -0.40944345590215175,
        "Trial B": -2.534190699225962,
        "Trial C": 2.2953815377854365
      },
      {
        "Trial A": -0.38648566207844004,
        "Trial B": -2.5187349009898745,
        "Trial C": 2.3065739021777856
      },
      {
        "Trial A": -0.3578427434294145,
        "Trial B": -2.490500932764758,
        "Trial C": 2.314350904364856
      },
      {
        "Trial A": -0.3375693010600844,
        "Trial B": -2.460097469254852,
        "Trial C": 2.320746621552868
      },
      {
        "Trial A": -0.3156315632601276,
        "Trial B": -2.4351635650238483,
        "Trial C": 2.3278956906556676
      },
      {
        "Trial A": -0.29626273838643313,
        "Trial B": -2.4085476812556115,
        "Trial C": 2.3459475165461043
      },
      {
        "Trial A": -0.27102635445539125,
        "Trial B": -2.383984848266357,
        "Trial C": 2.3534066335656054
      },
      {
        "Trial A": -0.2424110733577246,
        "Trial B": -2.3585826354013966,
        "Trial C": 2.3660631898470244
      },
      {
        "Trial A": -0.2159400974263677,
        "Trial B": -2.333190868723946,
        "Trial C": 2.375362953426709
      },
      {
        "Trial A": -0.19190574153028978,
        "Trial B": -2.305046330931228,
        "Trial C": 2.3842731231571133
      },
      {
        "Trial A": -0.17195094468104316,
        "Trial B": -2.2804024535212335,
        "Trial C": 2.3937977561724337
      },
      {
        "Trial A": -0.16334572804221031,
        "Trial B": -2.257849802752975,
        "Trial C": 2.4025170366262882
      },
      {
        "Trial A": -0.14484837663828834,
        "Trial B": -2.234767492455427,
        "Trial C": 2.4112503887649788
      },
      {
        "Trial A": -0.1282338177047786,
        "Trial B": -2.208685589002762,
        "Trial C": 2.421091032274613
      },
      {
        "Trial A": -0.11069246941534687,
        "Trial B": -2.187188613986582,
        "Trial C": 2.4286167240432848
      },
      {
        "Trial A": -0.0894263000031317,
        "Trial B": -2.164197702983212,
        "Trial C": 2.4385508269146907
      },
      {
        "Trial A": -0.07337305903143337,
        "Trial B": -2.149238805696672,
        "Trial C": 2.4521765764417487
      },
      {
        "Trial A": -0.047844738503507685,
        "Trial B": -2.127342472554162,
        "Trial C": 2.459203970520468
      },
      {
        "Trial A": -0.026634106618350258,
        "Trial B": -2.1068875700980527,
        "Trial C": 2.4697493143460365
      },
      {
        "Trial A": 0.007709927593753397,
        "Trial B": -2.0908343761329045,
        "Trial C": 2.4773792636546013
      },
      {
        "Trial A": 0.025810148450592214,
        "Trial B": -2.050348195929905,
        "Trial C": 2.4883030592150543
      },
      {
        "Trial A": 0.0387790104939954,
        "Trial B": -2.0222432703294935,
        "Trial C": 2.49882977368749
      },
      {
        "Trial A": 0.06135291084186913,
        "Trial B": -1.9795484676091586,
        "Trial C": 2.5074703091494603
      },
      {
        "Trial A": 0.08870770945655265,
        "Trial B": -1.9545308590722241,
        "Trial C": 2.515538473561887
      },
      {
        "Trial A": 0.10887884711978306,
        "Trial B": -1.9284665443991695,
        "Trial C": 2.5264585454550317
      },
      {
        "Trial A": 0.13803552837839675,
        "Trial B": -1.9069515172025733,
        "Trial C": 2.5355375553552313
      },
      {
        "Trial A": 0.15772673693959566,
        "Trial B": -1.8945977356277117,
        "Trial C": 2.5422609318159646
      },
      {
        "Trial A": 0.1814103360373364,
        "Trial B": -1.8664592764724706,
        "Trial C": 2.552165442095774
      },
      {
        "Trial A": 0.1951796157051821,
        "Trial B": -1.827908935235485,
        "Trial C": 2.563804693921568
      },
      {
        "Trial A": 0.21191340621836124,
        "Trial B": -1.7965677544295935,
        "Trial C": 2.574865730643066
      },
      {
        "Trial A": 0.2284738242182237,
        "Trial B": -1.7678959563886592,
        "Trial C": 2.5868244531081066
      },
      {
        "Trial A": 0.24036850900377738,
        "Trial B": -1.7436624499919384,
        "Trial C": 2.5950714327524214
      },
      {
        "Trial A": 0.26038307805409355,
        "Trial B": -1.7215872630237492,
        "Trial C": 2.608007525104347
      },
      {
        "Trial A": 0.2760821589439859,
        "Trial B": -1.6889853943062811,
        "Trial C": 2.623370106637543
      },
      {
        "Trial A": 0.30281083557394983,
        "Trial B": -1.6657977295682231,
        "Trial C": 2.6318445008849047
      },
      {
        "Trial A": 0.3247684403442249,
        "Trial B": -1.647683224339119,
        "Trial C": 2.640684179832888
      },
      {
        "Trial A": 0.3412385271334575,
        "Trial B": -1.6224986525495422,
        "Trial C": 2.6474529723129367
      },
      {
        "Trial A": 0.3561561653065134,
        "Trial B": -1.5911630083829755,
        "Trial C": 2.6616840589217943
      },
      {
        "Trial A": 0.37668456850392124,
        "Trial B": -1.5527099008884726,
        "Trial C": 2.6746098825930202
      },
      {
        "Trial A": 0.3947285244690777,
        "Trial B": -1.522862562103958,
        "Trial C": 2.6841320951899053
      },
      {
        "Trial A": 0.42113951472164707,
        "Trial B": -1.500014035807463,
        "Trial C": 2.6934039390644138
      },
      {
        "Trial A": 0.44532376035740157,
        "Trial B": -1.4713279555861152,
        "Trial C": 2.7025298044766886
      },
      {
        "Trial A": 0.4654273623090258,
        "Trial B": -1.431379395605881,
        "Trial C": 2.7123714455103505
      },
      {
        "Trial A": 0.4864201110649778,
        "Trial B": -1.3911922556234195,
        "Trial C": 2.723250362554841
      },
      {
        "Trial A": 0.5095464944996559,
        "Trial B": -1.3647891725712034,
        "Trial C": 2.736037528837007
      },
      {
        "Trial A": 0.5290522397150711,
        "Trial B": -1.336942297729629,
        "Trial C": 2.74725085884085
      },
      {
        "Trial A": 0.5473231989776981,
        "Trial B": -1.3009276864226813,
        "Trial C": 2.754833889501565
      },
      {
        "Trial A": 0.5744860259612212,
        "Trial B": -1.2769353559597267,
        "Trial C": 2.7595145608349565
      },
      {
        "Trial A": 0.6077521592648492,
        "Trial B": -1.2341237352525232,
        "Trial C": 2.7718835694648924
      },
      {
        "Trial A": 0.6466435282990876,
        "Trial B": -1.2020996821708874,
        "Trial C": 2.7829919511517067
      },
      {
        "Trial A": 0.6701156084055482,
        "Trial B": -1.1535316348034224,
        "Trial C": 2.794879663438985
      },
      {
        "Trial A": 0.6869050850059637,
        "Trial B": -1.1162232900959184,
        "Trial C": 2.8018862829552114
      },
      {
        "Trial A": 0.7443113525086926,
        "Trial B": -1.0756501925154367,
        "Trial C": 2.8109000376510087
      },
      {
        "Trial A": 0.7673774240871162,
        "Trial B": -1.046573609230521,
        "Trial C": 2.8192761987278034
      },
      {
        "Trial A": 0.7980316242623846,
        "Trial B": -1.0234750195531022,
        "Trial C": 2.829222187851645
      },
      {
        "Trial A": 0.8347358250485337,
        "Trial B": -0.9580866575739195,
        "Trial C": 2.8338971611003125
      },
      {
        "Trial A": 0.8667320724683842,
        "Trial B": -0.915172407090348,
        "Trial C": 2.8434671988872937
      },
      {
        "Trial A": 0.8959515471385678,
        "Trial B": -0.8785512567283753,
        "Trial C": 2.8568412943005157
      },
      {
        "Trial A": 0.9420534856943237,
        "Trial B": -0.8321122563720117,
        "Trial C": 2.86739217628038
      },
      {
        "Trial A": 0.9808974619593398,
        "Trial B": -0.770578371981685,
        "Trial C": 2.877220892798036
      },
      {
        "Trial A": 1.0158025102173758,
        "Trial B": -0.6993579659973792,
        "Trial C": 2.8921389155880335
      },
      {
        "Trial A": 1.046733464789217,
        "Trial B": -0.6555649512560784,
        "Trial C": 2.9048248336032847
      },
      {
        "Trial A": 1.104815221497215,
        "Trial B": -0.5934161227489734,
        "Trial C": 2.9145431941008764
      },
      {
        "Trial A": 1.1580823949331558,
        "Trial B": -0.5055378208896268,
        "Trial C": 2.9264916737478655
      },
      {
        "Trial A": 1.2365047063078867,
        "Trial B": -0.42283961260964165,
        "Trial C": 2.9329209524459343
      },
      {
        "Trial A": 1.3008279628235224,
        "Trial B": -0.3649028846931873,
        "Trial C": 2.9512483945295513
      },
      {
        "Trial A": 1.3629948424426541,
        "Trial B": -0.3074874869238261,
        "Trial C": 2.958755103974722
      },
      {
        "Trial A": 1.4474519669644412,
        "Trial B": -0.13227380088687055,
        "Trial C": 2.9697285032795766
      },
      {
        "Trial A": 1.6825926681962409,
        "Trial B": -0.018168011470989655,
        "Trial C": 2.9791366120633507
      },
      {
        "Trial A": 1.8223463605797108,
        "Trial B": 0.49285701072073534,
        "Trial C": 2.9920311162538815
      }
    ]
  },
  "encoding": {
    "x": {
      "field": "Trial A",
      "type": "quantitative"
    },
    "y": {
      "field": "Trial C",
      "type": "quantitative"
    }
  },
  "height": 600,
  "mark": "point",
  "width": 800
};
    var opt = {"renderer": "canvas", "actions": false};
    vegaEmbed("#A8183306fe917e1899f04d6633826bf78935eb3f10a5a31ad992722a3eabe4c87", spec, opt);
  </script>
  