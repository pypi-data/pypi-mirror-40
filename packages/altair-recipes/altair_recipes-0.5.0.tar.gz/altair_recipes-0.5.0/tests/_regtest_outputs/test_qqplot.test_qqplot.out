{
  "$schema": "https://vega.github.io/schema/vega-lite/v2.6.0.json",
  "config": {
    "view": {
      "height": 300,
      "width": 400
    }
  },
  "data": {
    "values": [
      {
        "Trial A": -2.9920805103614,
        "Trial B": -5.0074372044436,
        "Trial C": 2.0020181879054
      },
      {
        "Trial A": -1.6951221907376,
        "Trial B": -4.2568738665107,
        "Trial C": 2.005464791601
      },
      {
        "Trial A": -1.5498179320423,
        "Trial B": -3.9883575410347,
        "Trial C": 2.0138573967102
      },
      {
        "Trial A": -1.4529076525958,
        "Trial B": -3.8396239391795,
        "Trial C": 2.0261355922737
      },
      {
        "Trial A": -1.3668401791214,
        "Trial B": -3.7323886780125,
        "Trial C": 2.0374508434966
      },
      {
        "Trial A": -1.3061998629808,
        "Trial B": -3.6504110122301,
        "Trial C": 2.0473227297786
      },
      {
        "Trial A": -1.2389194047831,
        "Trial B": -3.5867701912527,
        "Trial C": 2.0618791637256
      },
      {
        "Trial A": -1.1966054746588,
        "Trial B": -3.4671282491245,
        "Trial C": 2.071590238427
      },
      {
        "Trial A": -1.1537647092188,
        "Trial B": -3.4092176349247,
        "Trial C": 2.0761637389907
      },
      {
        "Trial A": -1.1049025015175,
        "Trial B": -3.3546095121378,
        "Trial C": 2.0848846954091
      },
      {
        "Trial A": -1.0648325600541,
        "Trial B": -3.3074238921244,
        "Trial C": 2.0936565124268
      },
      {
        "Trial A": -1.0223539119203,
        "Trial B": -3.2442779004932,
        "Trial C": 2.1073172750515
      },
      {
        "Trial A": -0.9714518634046,
        "Trial B": -3.1781112399828,
        "Trial C": 2.1158585150958
      },
      {
        "Trial A": -0.9266128558619,
        "Trial B": -3.1133886478325,
        "Trial C": 2.1268056781698
      },
      {
        "Trial A": -0.880693881924,
        "Trial B": -3.0713694934276,
        "Trial C": 2.1395760521416
      },
      {
        "Trial A": -0.8397857514616,
        "Trial B": -3.0320933943718,
        "Trial C": 2.1623777548658
      },
      {
        "Trial A": -0.8102099687098,
        "Trial B": -2.9709534552505,
        "Trial C": 2.1697799129817
      },
      {
        "Trial A": -0.7773712117529,
        "Trial B": -2.9462078319728,
        "Trial C": 2.1787759472807
      },
      {
        "Trial A": -0.7410930650427,
        "Trial B": -2.9208735368032,
        "Trial C": 2.1888812891593
      },
      {
        "Trial A": -0.723233163556,
        "Trial B": -2.8525829790559,
        "Trial C": 2.1968728737048
      },
      {
        "Trial A": -0.693057944375,
        "Trial B": -2.8134353371086,
        "Trial C": 2.2098208649318
      },
      {
        "Trial A": -0.6703783744681,
        "Trial B": -2.7811356207844,
        "Trial C": 2.217028025025
      },
      {
        "Trial A": -0.6429539501052,
        "Trial B": -2.7441885031984,
        "Trial C": 2.2273335099798
      },
      {
        "Trial A": -0.6194790282575,
        "Trial B": -2.711605501586,
        "Trial C": 2.2392348886227
      },
      {
        "Trial A": -0.5984263790396,
        "Trial B": -2.6752193650029,
        "Trial C": 2.2506899500153
      },
      {
        "Trial A": -0.5683780045032,
        "Trial B": -2.6269905733007,
        "Trial C": 2.2628728285607
      },
      {
        "Trial A": -0.546093521944,
        "Trial B": -2.5874027307516,
        "Trial C": 2.2740283220472
      },
      {
        "Trial A": -0.5151322339619,
        "Trial B": -2.5548248330135,
        "Trial C": 2.2866092429424
      },
      {
        "Trial A": -0.4892050042718,
        "Trial B": -2.539207823859,
        "Trial C": 2.2985788278045
      },
      {
        "Trial A": -0.4692064590845,
        "Trial B": -2.5112341910823,
        "Trial C": 2.3086383369613
      },
      {
        "Trial A": -0.460217734731,
        "Trial B": -2.4760116792684,
        "Trial C": 2.3170757573935
      },
      {
        "Trial A": -0.4425934876127,
        "Trial B": -2.4423075907081,
        "Trial C": 2.3293399299621
      },
      {
        "Trial A": -0.4078742917348,
        "Trial B": -2.4086607098437,
        "Trial C": 2.3376528298815
      },
      {
        "Trial A": -0.3856711471279,
        "Trial B": -2.3673170472465,
        "Trial C": 2.3426387307202
      },
      {
        "Trial A": -0.3679451458001,
        "Trial B": -2.3384721351099,
        "Trial C": 2.3570841935436
      },
      {
        "Trial A": -0.3482764497456,
        "Trial B": -2.3160299514076,
        "Trial C": 2.3639840992611
      },
      {
        "Trial A": -0.3313052435431,
        "Trial B": -2.2922042562748,
        "Trial C": 2.3721458362989
      },
      {
        "Trial A": -0.3154275221856,
        "Trial B": -2.274598889758,
        "Trial C": 2.3818889009658
      },
      {
        "Trial A": -0.3017057927779,
        "Trial B": -2.2595162738401,
        "Trial C": 2.3889445601081
      },
      {
        "Trial A": -0.2760342857019,
        "Trial B": -2.2398577995756,
        "Trial C": 2.4026795950383
      },
      {
        "Trial A": -0.2613064138201,
        "Trial B": -2.2224121416088,
        "Trial C": 2.4101534402488
      },
      {
        "Trial A": -0.2251000606563,
        "Trial B": -2.2090950800718,
        "Trial C": 2.4221905239342
      },
      {
        "Trial A": -0.1895023803109,
        "Trial B": -2.1713533239694,
        "Trial C": 2.4277715639213
      },
      {
        "Trial A": -0.1623353503592,
        "Trial B": -2.152709477204,
        "Trial C": 2.4398622860662
      },
      {
        "Trial A": -0.1427847054602,
        "Trial B": -2.135297497953,
        "Trial C": 2.4509549925732
      },
      {
        "Trial A": -0.1319870171962,
        "Trial B": -2.1033281086819,
        "Trial C": 2.4597923158295
      },
      {
        "Trial A": -0.1075857228599,
        "Trial B": -2.0912979752517,
        "Trial C": 2.4692166156414
      },
      {
        "Trial A": -0.0771975040102,
        "Trial B": -2.0654971083978,
        "Trial C": 2.4801561140508
      },
      {
        "Trial A": -0.0569373221733,
        "Trial B": -2.0417952084377,
        "Trial C": 2.4904201216983
      },
      {
        "Trial A": -0.0363972114194,
        "Trial B": -2.0128192447018,
        "Trial C": 2.5013354956166
      },
      {
        "Trial A": -0.0226126472017,
        "Trial B": -1.9743041075266,
        "Trial C": 2.5098218324693
      },
      {
        "Trial A": -0.0084872458623,
        "Trial B": -1.9520861970742,
        "Trial C": 2.5190975670026
      },
      {
        "Trial A": 0.005105247024,
        "Trial B": -1.9219785755528,
        "Trial C": 2.525944668481
      },
      {
        "Trial A": 0.0228575259075,
        "Trial B": -1.9032766796999,
        "Trial C": 2.5349739158913
      },
      {
        "Trial A": 0.0505629382563,
        "Trial B": -1.888020293478,
        "Trial C": 2.542090787821
      },
      {
        "Trial A": 0.0723499973981,
        "Trial B": -1.8580722942157,
        "Trial C": 2.5588466239676
      },
      {
        "Trial A": 0.0869773706171,
        "Trial B": -1.8427282464013,
        "Trial C": 2.568310156065
      },
      {
        "Trial A": 0.1046016863252,
        "Trial B": -1.8131908183145,
        "Trial C": 2.5750890510918
      },
      {
        "Trial A": 0.1247524737045,
        "Trial B": -1.778144753212,
        "Trial C": 2.5889596217594
      },
      {
        "Trial A": 0.1462222420997,
        "Trial B": -1.7538712636843,
        "Trial C": 2.5981060494326
      },
      {
        "Trial A": 0.1640281645277,
        "Trial B": -1.7277847742916,
        "Trial C": 2.6030092266291
      },
      {
        "Trial A": 0.1818524767952,
        "Trial B": -1.7121282553481,
        "Trial C": 2.61184720489
      },
      {
        "Trial A": 0.1960704421625,
        "Trial B": -1.686614858129,
        "Trial C": 2.6226877814513
      },
      {
        "Trial A": 0.2258438623085,
        "Trial B": -1.6491812257244,
        "Trial C": 2.634975254109
      },
      {
        "Trial A": 0.2538835840704,
        "Trial B": -1.6067824672965,
        "Trial C": 2.6474238922223
      },
      {
        "Trial A": 0.2760687970761,
        "Trial B": -1.5789568726029,
        "Trial C": 2.656053948603
      },
      {
        "Trial A": 0.2997276447832,
        "Trial B": -1.5506787563166,
        "Trial C": 2.6723421559565
      },
      {
        "Trial A": 0.3218354252783,
        "Trial B": -1.527973318725,
        "Trial C": 2.6802084335215
      },
      {
        "Trial A": 0.3350588769535,
        "Trial B": -1.4921881165513,
        "Trial C": 2.6887363633922
      },
      {
        "Trial A": 0.3586568079675,
        "Trial B": -1.4675510621696,
        "Trial C": 2.6962486347182
      },
      {
        "Trial A": 0.3995464566727,
        "Trial B": -1.4314753137793,
        "Trial C": 2.7060546636233
      },
      {
        "Trial A": 0.4299079951034,
        "Trial B": -1.3929732948729,
        "Trial C": 2.7159831430806
      },
      {
        "Trial A": 0.4520663478711,
        "Trial B": -1.3797662364045,
        "Trial C": 2.7252450053925
      },
      {
        "Trial A": 0.4800283037093,
        "Trial B": -1.3588253073514,
        "Trial C": 2.7389894917536
      },
      {
        "Trial A": 0.5151435648825,
        "Trial B": -1.3199864893496,
        "Trial C": 2.7541403248275
      },
      {
        "Trial A": 0.5294603422159,
        "Trial B": -1.2863576414118,
        "Trial C": 2.7637811924923
      },
      {
        "Trial A": 0.5594006710397,
        "Trial B": -1.2600808477399,
        "Trial C": 2.7708904542472
      },
      {
        "Trial A": 0.5874733667988,
        "Trial B": -1.2238337691876,
        "Trial C": 2.7756742993315
      },
      {
        "Trial A": 0.6096866844942,
        "Trial B": -1.1884820490508,
        "Trial C": 2.7845852727687
      },
      {
        "Trial A": 0.6543470870591,
        "Trial B": -1.1494359962851,
        "Trial C": 2.7931385742879
      },
      {
        "Trial A": 0.6890560825685,
        "Trial B": -1.1079281334903,
        "Trial C": 2.7998078765308
      },
      {
        "Trial A": 0.7086230290895,
        "Trial B": -1.0728355633933,
        "Trial C": 2.8113518464576
      },
      {
        "Trial A": 0.7261622457209,
        "Trial B": -1.0347290154209,
        "Trial C": 2.8212503755652
      },
      {
        "Trial A": 0.7566602570888,
        "Trial B": -0.9895378948866,
        "Trial C": 2.8326114468749
      },
      {
        "Trial A": 0.7975445911422,
        "Trial B": -0.9461811950496,
        "Trial C": 2.8423053149126
      },
      {
        "Trial A": 0.8384965600288,
        "Trial B": -0.9243071607381,
        "Trial C": 2.8538654130704
      },
      {
        "Trial A": 0.8663145914674,
        "Trial B": -0.8801863753057,
        "Trial C": 2.8660398935476
      },
      {
        "Trial A": 0.8977665751805,
        "Trial B": -0.8338467443887,
        "Trial C": 2.8743184481461
      },
      {
        "Trial A": 0.9258485479124,
        "Trial B": -0.7939946719407,
        "Trial C": 2.8823583178727
      },
      {
        "Trial A": 0.9559979103903,
        "Trial B": -0.7420228877088,
        "Trial C": 2.8922019722188
      },
      {
        "Trial A": 1.0114002209511,
        "Trial B": -0.6767309956691,
        "Trial C": 2.9026564949192
      },
      {
        "Trial A": 1.0910802155496,
        "Trial B": -0.6234492255437,
        "Trial C": 2.9112753615682
      },
      {
        "Trial A": 1.160798328214,
        "Trial B": -0.5682621413668,
        "Trial C": 2.9220948534374
      },
      {
        "Trial A": 1.2148714937602,
        "Trial B": -0.4733120264371,
        "Trial C": 2.9297455924767
      },
      {
        "Trial A": 1.2603310062532,
        "Trial B": -0.4234423732345,
        "Trial C": 2.9379212381745
      },
      {
        "Trial A": 1.3320555294181,
        "Trial B": -0.365540572812,
        "Trial C": 2.9517870048054
      },
      {
        "Trial A": 1.4194211580513,
        "Trial B": -0.2878372951454,
        "Trial C": 2.9571627751086
      },
      {
        "Trial A": 1.4876092912207,
        "Trial B": -0.1551160984178,
        "Trial C": 2.9669236822861
      },
      {
        "Trial A": 1.6631020273791,
        "Trial B": 0.008553006938,
        "Trial C": 2.9765755671108
      },
      {
        "Trial A": 1.9309643128605,
        "Trial B": 0.3002428123032,
        "Trial C": 2.9902008467158
      }
    ]
  },
  "encoding": {
    "x": {
      "field": "Trial A",
      "type": "quantitative"
    },
    "y": {
      "field": "Trial C",
      "type": "quantitative"
    }
  },
  "height": 600,
  "mark": "point",
  "width": 800
}